app/persistence/fabric/CRUDService.ts

```diff
@@ -540,6 +564,22 @@ export class CRUDService {
 	}
 	// Orderer BE-303
 
+	/**
+	 *
+	 *
+	 * @param {*} blockHeight
+	 * @returns
+	 * @memberof CRUDService
+	 */
+	async isBlockAvailableInDB(channel_genesis_hash: string, blockHeight: any) {
+		const count: any = await this.sql.getRowsBySQlCase(
+			`SELECT COUNT(*) FROM blocks WHERE channel_genesis_hash= $1 AND blocknum = $2`,
+			[channel_genesis_hash, blockHeight]
+		);
+		console.log(count.count);
```

Comment:
Can you please remove the console log here.

Reply:
removed.